Libraries/LibWeb/WebGL/WebGLRenderingContextImpl.cpp

```diff
@@ -1,14 +1,16 @@
 /*
  * Copyright (c) 2024-2025, Aliaksandr Kalenik <kalenik.aliaksandr@gmail.com>
  * Copyright (c) 2024-2025, Luke Wilde <luke@ladybird.org>
+ * Copyright (c) 2025, Undefine <undefine@undefine.pl>
  *
  * SPDX-License-Identifier: BSD-2-Clause
  */
 
 #define GL_GLEXT_PROTOTYPES 1
-#include <GLES2/gl2.h>
-#include <GLES2/gl2ext.h>
+
+#include <GLES3/gl3.h>
 extern "C" {
+#include <GLES2/gl2ext.h>
 #include <GLES2/gl2ext_angle.h>
 }
```

Comment:
why do we switch from gl2 to gl3 with this change? could you add some explanation in the commit description or even move into separate commit?